src/Microsoft.PowerShell.Commands.Management/Microsoft.PowerShell.Commands.Management.csproj

```diff
@@ -22,7 +22,6 @@
     <Compile Remove="commands\management\GetClipboardCommand.cs" />
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Verify approved aliases list.All present Cmdlets should have the correct ConfirmImpact
```powershell
Expected $null or empty, but got @(@{Name=Get-HotFix; ConfirmImpact=; SideIndicator==>}, @{Name=Tee-Object; ConfirmImpact=None; SideIndicator=<=}).
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\DefaultCommands.Tests.ps1: line 604
604:         $result | Should -BeNullOrEmpty
```